src/components/ScrollButton.vue

```diff
@@ -9,7 +9,6 @@
     <svg
       viewBox="0 0 24 24"
       xmlns="http://www.w3.org/2000/svg"
-      role="img"
```

Comment:
Since the icon is hidden, it is not necessary to give it the `img` role[^1]. If we are going to treat the icon as an image, we must give it a title and (optionally) a description.

[^1]: https://css-tricks.com/accessible-svg-icons/